tools/project_manager/update_illos.php

```diff
@@ -0,0 +1,206 @@
+<?php
+// Replace an illustration file.
+
+$relPath = '../../pinc/';
+include_once($relPath.'base.inc');
+include_once($relPath.'theme.inc');
+include_once($relPath.'Project.inc');
+include_once($relPath.'misc.inc'); // get_upload_err_msg, get_enumerated_param, attr_safe, html_safe
+include_once($relPath.'project_states.inc'); // PROJ_NEW
+
+require_login();
+
+// (This script's functionality overlaps that of handle_bad_page.php.
+// They should perhaps be refactored.)
+
+$projectid = validate_projectID('projectid', @$_REQUEST['projectid']);
+
+$image     = @$_REQUEST['image'];
+// Note that using validate_page_image_filename() here would be inappropriate,
+// because it's specific to *page* images (which this isn't),
+// and so enforces a maximum filename length.
+if (!preg_match('/^\w[\w.-]*\.(png|jpg)$/', $image)) // see _check_file() in add_files.php
+{
+    // This should only happen if someone is URL-tweaking.
+    die(_("Parameter 'image' is not a valid illustration filename."));
+}
+
+$project = new Project($projectid);
+$operation = get_enumerated_param($_REQUEST, 'operation', 'replace', array('replace', 'delete'));
+
+$operation_image_str;
+if ($operation == 'replace') {
+    $operation_image_str = _('Replace Illustration');
+} else {
+    $operation_image_str = _('Delete Illustration');
+}
+
+output_header("$operation_image_str: {$project->nameofwork}");
```

Comment:
Let's make this page mirror handle_bad_page a bit more:

* Make the HTML header and `<h1>` title `_("Update Illustration")`.
* Add a small block below this that give some project info, including the illustration being updated (ala https://github.com/DistributedProofreaders/dproofreaders/blob/master/tools/project_manager/handle_bad_page.php#L81)
* The `<h2>` can then be `$operation_image_str`

Reply:
Here is what it looks like now:
![image](https://user-images.githubusercontent.com/1526781/76916929-04016100-6890-11ea-8f17-3adf137a6a24.png)
![image](https://user-images.githubusercontent.com/1526781/76916930-05cb2480-6890-11ea-8af6-75f7b49ba2e2.png)